### PR TITLE
remove further uses of default_storage (bug 1167271)

### PR DESCRIPTION
**Pull request by washort**
(sto)rage against the machine

### CODE REVIEW

File: mkt/files/models.py [hunk 1]
```diff
@@ -231,7 +230,7 @@ def cleanup_file(sender, instance, **kw):
             return
         # FIXME: This needs to check if the file is public or not and choose
         # the appropriate storage backend.
-        if filename and storage.exists(filename):
+        if filename and public_storage.exists(filename):
```
Comment (robhudson) [inline mkt/files/models.py]: I think this should just check both public and private (and remove the FIXME comment)

Comment (washort) [inline mkt/files/models.py]: good point, fixed

File: mkt/files/models.py [hunk 2]
```diff
@@ -229,12 +228,12 @@ def cleanup_file(sender, instance, **kw):
...
-        # FIXME: This needs to check if the file is public or not and choose
-        # the appropriate storage backend.
-        if filename and storage.exists(filename):
+        if filename and (public_storage.exists(filename) or
+                         private_storage.exists(filename)):
             log.info('Removing filename: %s for file: %s'
                      % (filename, instance.pk))
-            storage.delete(filename)
+            public_storage.delete(filename)
+            private_storage.delete(filename)
```
Comment (robhudson) [inline mkt/files/models.py]: Will this throw an error if it exists on one but not the other?

File: mkt/site/utils.py [hunk 3]
```diff
@@ -265,11 +264,11 @@ def __iter__(self):
             return iter([])
 
         chunk = 4096
-        fp = storage.open(self.path, 'rb')
+        fp = private_storage.open(self.path, 'rb')
         if 'wsgi.file_wrapper' in self.request.META:
             return self.request.META['wsgi.file_wrapper'](fp, chunk)
         else:
-            self['Content-Length'] = storage.size(self.path)
+            self['Content-Length'] = private_storage.size(self.path)
```
Comment (robhudson) [inline mkt/site/utils.py]: I wonder if this should be `local_storage` here after my patch. `HttpResponseSendFile` should never be called if we are using S3.

File: mkt/submit/tests/test_views.py [hunk 4]
```diff
@@ -16,7 +15,7 @@
 from mkt.files.tests.test_models import UploadTest as BaseUploadTest
 from mkt.reviewers.models import EscalationQueue
 from mkt.site.fixtures import fixture
-from mkt.site.storage_utils import copy_to_storage
+from mkt.site.storage_utils import public_storage, copy_to_storage
```
Comment (robhudson) [inline mkt/submit/tests/test_views.py]: c < p

**Comment by robhudson** [comment]
r+wc

File: mkt/files/models.py (hunk 2 above)
Comment (washort) [inline mkt/files/models.py]: Docs say delete does not throw exceptions when the path doesn't exist.

File: mkt/site/utils.py (hunk 3 above)
Comment (washort) [inline mkt/site/utils.py]: OK.

File: mkt/files/models.py (hunk 2 above)
Comment (robhudson) [inline mkt/files/models.py]: Ok cool

**Pull request merged**